flixel/util/FlxStringUtil.hx

```diff
@@ -262,14 +262,8 @@ class FlxStringUtil
 	 */
 	public static function getDomain(url:String):String
```

Comment:
Considering even the old function was good enough for years / nobody reported any issues with it thus far, I don't think there's a need to support every edge case. :)